packages/mds-agency/request-handlers.ts

```diff
@@ -218,9 +218,16 @@ export const updateVehicle = async (req: AgencyApiUpdateVehicleRequest, res: Age
   }
 }
 
-const uuidSchema = { type: 'string', format: 'uuid' }
-const numberSchema = { type: 'number' }
-const timestampSchema = { type: 'integer', minimum: 100_000_000_000, maximum: 99_999_999_999_999 }
```

Comment:
Feels a little funny to define the allowable date range as March 1973 - November 5138 but it make sense as a guard against invalid values, one some level.

Reply:
The range is just that which gets interpreted as `ms`... detects if seconds or nanoseconds are supplied.